raven_statsd/recipes/grafana.rb

```diff
@@ -23,17 +8,39 @@
 template "/etc/grafana/grafana.ini" do
 	source "grafana.ini.erb"
 	variables ({
-			:google_client_id => node[:raven_statsd][:google][:client_id],
-			:google_client_secret => node[:raven_statsd][:google][:client_secret],
-			:google_domain => node[:raven_statsd][:google][:domain],
-			:admin_username => node[:raven_statsd][:admin][:username],
-			:admin_password => node[:raven_statsd][:admin][:password],
-			:root_url => node[:raven_statsd][:server][:root_url],
-			:data_dir => node[:raven_statsd][:grafana][:data_dir]
-			})
+		:data_dir => node[:raven_statsd][:grafana][:data_dir],
+		:fqdn => node[:raven_statsd][:grafana][:fqdn],
+		:public_fqdn => node[:raven_statsd][:grafana][:public_fqdn],
+		:http_port => node[:raven_statsd][:grafana][:http_port],
+		:admin_username => node[:raven_statsd][:grafana][:admin][:username],
+		:admin_password => node[:raven_statsd][:grafana][:admin][:password],
+		:google_client_id => node[:raven_statsd][:grafana][:google][:client_id],
+		:google_client_secret => node[:raven_statsd][:grafana][:google][:client_secret],
+		:google_domain => node[:raven_statsd][:grafana][:google][:domain]
+	})
 	notifies :restart, "service[grafana-server]", :delayed
 end
 
 service "grafana-server" do
 	action [ :start, :enable ]
 end
+
+my_auth = Base64.encode64("#{node[:raven_statsd][:grafana][:admin][:username]}:#{node[:raven_statsd][:grafana][:admin][:password]}")
+
+http_request 'create_influxdb_data_source' do
```

Comment:
I need to add a not-if that sees if this already exists thats going to be a bit of a pain